### DISCUSSION TITLE
Can it accept the API requests similar to the Google reCAPTCHA API?

### DISCUSSION THREAD

**Discussion by GamePlayer-8** [librecaptcha/lc-core#199] [Q&A]
Heyo,

As the title says - can it accept the API requests similar to the reCAPTCHA ones?

**Comment by hrj** [comment]
Hi,

We haven't tried to match with reCAPTCHA API yet, but that's a good idea.

I may not be able to work on it immediately, but contributions are welcome!

**Comment by GamePlayer-8** [comment]
Would be really cool to bind it. I'm hosting the instance on https://captcha.xaviama.dank-me.me, but I'd like to connect with Matrix etc.

**Comment by vinceh121** [comment]
The problem with making a reCAPTCHA compatible API, is that most clients doing server side validation won't allow to point the API requests to a custom server.
